frontend/web/components/pages/AccountSettingsPage.js

```diff
@@ -126,6 +128,7 @@ class TheComponent extends Component {
   render() {
     const {
       state: {
+        auth_type,
```

Comment:
Using snake case here, since first_name, last_name and similar fields are also doing the same for fields fetched from the API

Reply:
Ok